homeassistant/components/intellifire/manifest.json

```diff
@@ -3,12 +3,8 @@
   "name": "IntelliFire",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/intellifire",
-  "requirements": [
-    "intellifire4py==0.4.3"
-  ],
+  "requirements": ["intellifire4py==0.5"],
```

Comment:
I found a bug in the HTML returned by the fireplace in that it doesn't have proper MIME typing so in my module I changed from:

```python
                    json_data = await response.json()
```

to 

```python
                    json_data = await response.json(content_type=None)
```

which allows for improper mime-typed JSON data

Reply:
@epenet <-- here is the info about the dependency bump -> I'll move this to the PR description

Reply:
That's a description, it is not a link.
You should add the link to the Github comparison: https://github.com/jeeftor/intellifire4py/compare/0.4.3...0.5

Reply:
Thanks! - added to description